trulens_eval/trulens_eval/feedback/prompts.py

```diff
@@ -219,6 +219,56 @@
 {response}
 """
 
+"""
```

Comment:
This prompt seems to be missing its variable assignment?

Reply:
thanks; that was an old one I was experimenting with. Removed it